### DISCUSSION TITLE
Thermal runaway with PID auto-tuning

### DISCUSSION THREAD

**Discussion by G4GNX** [Jyers/Marlin#508] [Q&A] [answered]
I compiled a new firmware today from 1.3 and just added in some of my fixed offsets for convenience.
I tried the PID auto-tune from the menu and started by setting the hot-end temp to 200. All appeared to work, so I set the bed temp at 60 and started a bed PID tune. Whilst it was running, I could hear a fan starting and stopping and I think it was the part cooling fan. Before the bed tune had completed, the printer threw a Thermal Runaway error and I had to re-start it, so lost the tuning values. I tried another hot-end tune at 180, then a bed tune at 50 and all was OK. I subsequently ran two more PIDs for hot-end and bed at 200/60 and they completed OK. I've since done a 1st layer print at 200/60 and that was OK.
I've never had a thermal runaway before, so I presume this is a glitch in the firmware and maybe needs some investigation.
I'm quite happy to supply my version of configuration.h and configuration_adv.h

**Comment by Jyers** [comment, marked as answer]
An interesting occurrence for sure, but it was most likely a one off. The firmware doesn't do anything differently than you would do from the console. It simply executes a pid autotune gcode command the same you would do from the terminal. If marlin threw a thermal runaway warning either something hardware wise messed up, or it was to do with some kind of bug in base marlin. Either way, there not much I can do on my side of things. If you see it again I might suggest opening an issue on the main marlin repo.

**Reply by G4GNX** [reply]
A fair comment. :-)
I guess it's just coincidence. ISTR your default setting was 180/50 and I changed to 200/600.
I think it may be a base issue in Marlin, as I've had no issues since and have printed a few things.
I'll wait to see if it happens again before reporting to Marlin direct.
